terraform/role.yaml

```diff
@@ -0,0 +1,21 @@
+kind: role
+metadata:
+  name: terraform
+spec:
+  allow:
+    rules:
+      - resources:
+        - user
+        - role
+        - token
+        - trusted_cluster
+        - github
+        - oidc
+        - saml
+        - app
+        - database
+        - session_recording_config
+        - cluster_networking_config
+        - cluster_auth_preference
+        verbs: ['list','create','read','update','delete']
+version: v3
```

Comment:
v4?

Reply:
Yes, this is something I used for debug, removed the file.